engine/schema/src/main/java/com/cloud/network/dao/NetworkDetailsDao.java

```diff
@@ -16,10 +16,26 @@
 // under the License.
 package com.cloud.network.dao;
 
+import java.util.Map;
+import java.util.List;
+
 import org.apache.cloudstack.resourcedetail.ResourceDetailsDao;
 
 import com.cloud.utils.db.GenericDao;
 
 public interface NetworkDetailsDao extends GenericDao<NetworkDetailVO, Long>, ResourceDetailsDao<NetworkDetailVO> {
```

Comment:
could extend `ResourceDetailsDaoBase` instead of `ResourceDetailsDao`

Reply:
`ResourceDetailsDaoBase` is an abstract class, it can not be parent of an interface. Shall I make NetworkDetailsDao abstract class as well? What do you think?

Reply:
@soreana 
sorry for the mistake. I just noticed NetworkDetailsDaoImpl extends ResourceDetailsDaoBase already.

Reply:
Thanks for checking 🙏